utils/stellar.js

```diff
@@ -60,6 +61,68 @@ exports.stellar = (function() {
     })
   }
 
+  createEscrowAccount = (secretKey, shaSignerKey, callback) => {
+    StellarSdk.Network.useTestNetwork();
+    let sourceKeypair = StellarSdk.Keypair.fromSecret(secretKey);
+    let escrowKeypair = StellarSdk.Keypair.random();
+    
+    (async () => {
+      try{
+        
+        const account = await stellarServer.loadAccount(sourceKeypair.publicKey())
+        let transaction = new StellarSdk.TransactionBuilder(account)
+        .addOperation(StellarSdk.Operation.createAccount({
+          destination: escrowKeypair.publicKey(),
+          startingBalance: '100'
+        })).build();
+        transaction.sign(sourceKeypair);
+        stellarServer.submitTransaction(transaction).then(() => {
+          return stellarServer.loadAccount(escrowKeypair.publicKey())
+        }).then((escrowAccount) => {
+          let transaction = new StellarSdk.TransactionBuilder(escrowAccount)
+          .addOperation(StellarSdk.Operation.setOptions({
+            signer: {sha256Hash: shaSignerKey, weight: 2},
+            masterWeight: 4,
+            lowThreshold: 2,
+            medThreshold: 2,
+            highThreshold: 2
+          })).build();
+          transaction.sign(escrowKeypair)
+          return stellarServer.submitTransaction(transaction)
+        }).then(() => {
+          callback(escrowKeypair)
+        })
+      }catch(e){
+        console.log('error: ', e)
+      }
+    })();
+    
+  }
+
+  acceptPayment = (shaPreimage, escrowAccountKey, myAccountId) => {
+    StellarSdk.Network.useTestNetwork();
+    (async () => {
```

Comment:
If you went with creating named function expressions, then you could do that here as well